setup.sh

```diff
@@ -298,13 +298,21 @@ function synchronize_files()
   setup_config_file
   ASSERT_IF_NOT_EQ_ZERO "Config file failed" "$?"
 
-  if [ -f "$HOME/.bashrc" ]; then
-      # Add to bashrc
-      echo "# $app_name" >> "$HOME/.bashrc"
-      echo "source $libdir/$BASH_AUTOCOMPLETE.sh" >> "$HOME/.bashrc"
-      update_path
+  # Add tabcompletion to bashrc
+  if [[ -f "$HOME/.bashrc"  ||  -h "$HOME/.bashrc" ]]; then
+    append_bashcompletion ".bashrc"
+    update_path
   else
-      warning "Unable to find a shell."
+    warning "Unable to find a .bashrc file."
+  fi
+
+  # Add tabcompletion to zshrc
+  if [[ -f "$HOME/.zshrc" ||  -h "$HOME/.zshrc" ]]; then 
+    echo "# Enable bash completion for zsh" >> "$HOME/.zshrc"
```

Comment:
Please, correct me if I'm wrong, but looks like that `setup.sh` will trigger a warning in a pure zsh system, right? Or zsh also relies on `.bashrc`? Maybe you want an `elif` here, e.g,:

```
if [[ -f "$HOME/.bashrc" || -h "$HOME/.bashrc" ]]; then
...
elif [[ -f "$HOME/.zshrc" || -h "$HOME/.zshrc" ]]; then
...
else
 complain about something
fi
```

Reply:
You are correct, it would also complain in a pure bash system if it couldn't find a .zshrc file. 
My idea was to complain in both cases just to let the user know that it wasn't able to add tabcompletion, maybe this could explicitly mention the tabcompletions.
I use mainly zsh but I also have bash and would like to have tabcompletions for both, I wouldn't like  if the setup.sh silently failed to install tab completion to one of my shells.
Your suggestion would also cause the tabcompletions to be installed for bash but never for zsh if a bashrc file is present for the user, so I believe we should keep it as is. What do you think?

Reply:
Hi @alan-barzilay 
I see your point.
However, I'll feel annoyed if I see zsh warnings in my bash system. In my case, I'm not a zsh user; what is the point of showing me such a warning? Other users might feel confused and think that maybe their installation did not complete correctly. Perhaps you can try to alleviate this issue by checking if the users have zsh installed in their system before sending a warning message.

Reply:
@rodrigosiqueira We could create a better message to let the user know that this is not necessarily an error, but I agree that directly checking if the user has zsh installed is a better approach. 
Do you know of a way to easily confirm if the user has installed a given program? I couldn't think of a good and general way of doing this

Reply:
Maybe something like:

`[ -x "$(command -v zsh)" ]`

Reply:
nice! never thought of using command this way, done